code/modules/ai/ai_mind.dm

```diff
@@ -0,0 +1,4 @@
+//A mind that houses a personality and attitudes
+//Influences decisions based on node weights and later on, ability activations (xenomorphs and humans)
+
+/datum/ai_mind
```

Comment:
does this still need to exist now

Reply:
Ai mind having a actual purpose implemented